File: Programmers/Level2/큰 수 만들기/wanderer-s.js
function solution(number, k) {
  let biggestNumArr = []
  
  for(const num of number) {
    while ( k > 0 && biggestNumArr && biggestNumArr[biggestNumArr.length - 1] < num) {
      k--
      biggestNumArr.pop()
    }
    
    biggestNumArr.push(num)
  }

  return biggestNumArr.slice(0, biggestNumArr.length - k).join('')
}

File: Leetcode/Easy/844. Backspace String Compare/wanderer-s.py
class Solution:
  def backspaceCompare(self, s: str, t: str) -> bool:
    s_result = []
    t_result = []
    
    for char in s:
      if s_result and char == '#':
        s_result.pop()
      elif char !='#':
        s_result.append(char)
      
    for char in t:
      if t_result and char == '#':
        t_result.pop()
      elif char !='#':
        t_result.append(char)
        
    s_result = ''.join(s_result)
    t_result = ''.join(t_result)
    
    return s_result == t_result
    